src/main/java/teammates/common/util/Const.java

```diff
@@ -386,7 +386,7 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String FEEDBACK_QUESTION_GETLINK =
                 "Get a submission link to this particular question. "
                 + "Useful if you want students to answer individual questions separately or at different points in time.";
-        public static final String FEEDBACK_QUESTION_CANCEL = "Discard your changes";
+        public static final String FEEDBACK_QUESTION_DISCARDCHANGES = "Discard your changes";
```

Comment:
Currently we are using 'you' when displaying status messages to users and for UI elements.

However, there are some usages of 'I':
 - in relation to results publishing date: 'I want to manually publish the results.'
 - and the feedback path I'm working on may also use 'I'
 - the sample questions also uses I (e.g., my feedback on my teammates)

I think we should standardize to use 'you' as much as possible